src/bidiMapper/modules/context/BrowsingContextImpl.ts

```diff
@@ -275,6 +275,8 @@ export class BrowsingContextImpl {
 
   async getOrCreateSandbox(sandbox: string | undefined): Promise<Realm> {
     if (sandbox === undefined || sandbox === '') {
+      await this.#lifecycle.DOMContentLoaded;
```

Comment:
So the `this.#defaultRealmDeferred` is created for the `about:blank`, and is returned instead? What if I open an `about:blank` page? This would make sense to cover with e2e

Reply:
Yes, `this.#defaultRealmDeferred` is resolved once for the `about:blank` and reset after navigation.
Please see the (updated) description for the event.